x-pack/test/reporting/api/generate/csv_saved_search.ts

```diff
@@ -268,6 +269,68 @@ export default function({ getService }: { getService: any }) {
 
         await esArchiver.unload('reporting/hugedata');
       });
+
+      it('for docvalue_fields', async () => {
+        // load test data that contains a saved search and documents
+        await esArchiver.load('reporting/ecommerce');
+        await esArchiver.load('reporting/ecommerce_kibana');
+
+        const params = {
```

Comment:
This is a copy of the POST body that I captured from a network inspector after running the use case through a browser.